src/renderer/features/playlists/components/playlist-detail-content.tsx

```diff
@@ -156,6 +157,18 @@ export const PlaylistDetailContent = ({ tableRef }: PlaylistDetailContentProps)
 
     const loadMoreRef = useRef<HTMLButtonElement | null>(null);
 
+    // Duplicates are only present if on Navidrome
+    const getId = useCallback(
+        (data: GetRowIdParams<any>): string => {
+            return server?.type === ServerType.JELLYFIN
+                ? data.data.id
+                : `${data.data.id}-${data.data.pageIndex}`;
+        },
+        [server?.type],
+    );
```

Comment:
Does it affect performance to just use `${data.data.id}-${data.data.pageIndex}` as the global id instead of the conditional? If not, might be cleaner to just have one implementation rather than two.

Reply:
Looks like `getRowNode` is generally just a linear search underneath, so using I was hoping it'd be a more efficient implementation, but no. So I'll just reimplement using `forEachNode`, which is equally (or more, for multiple items) efficient currently.